google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -160,6 +160,7 @@ def _get_shards(gcs_bucket_name: str, gcs_prefix: str) -> List[documentai.Docume
     for byte in byte_array:
         shards.append(documentai.Document.from_json(byte, ignore_unknown_fields=True))
 
+    shards.sort(key=lambda x: int(x.shard_info.shard_index))
```

Comment:
Does every document.proto had a shard index even if it's not sharded ? What will happen if the document that's passed in isn't sharded?

Reply:
Unsharded documents won't have a shard index, but it will have `shard_info`.

`int(x.shard_info.shard_index)` will evaluate to `0`. Plus if there's only one shard, sorting won't do anything. Would you prefer I add a condition to check if it is multisharded first?

Reply:
Yea i think validating that it's multiple would be better.